tests/Load/getUser.js

```diff
@@ -0,0 +1,41 @@
+import http from 'k6/http';
+import counter from 'k6/x/counter';
+
+import ScenarioUtils from './utils/scenarioUtils.js';
+import Utils from './utils/utils.js';
+import InsertUsersUtils from "./utils/insertUsersUtils.js";
+
+const scenarioName = 'getUser';
+
+const utils = new Utils();
+const scenarioUtils = new ScenarioUtils(utils, scenarioName);
+const insertUsersUtils = new InsertUsersUtils(utils, scenarioName);
+
+insertUsersUtils.execInsertUsersCommand();
+const users = insertUsersUtils.getInsertedUsers();
+
+export function setup() {
+    return {
+        users: users,
+    };
+}
+
+export const options = scenarioUtils.getOptions();
+
+export default function getUser(data) {
+    const user = data.users[counter.up()];
+    utils.checkUserIsDefined(user);
+
+    const {id} = user;
```

Comment:
const { id } = users
need to add spaces in cases like this, it will be easier to understand the code